envoy/accesslog.cc

```diff
@@ -54,18 +54,18 @@ AccessLog::AccessLog(std::string path) : path_(path), fd_(-1), open_count_(1) {}
 
 AccessLog::~AccessLog() {}
 
-void AccessLog::Entry::InitFromRequest(std::string listener_id,
-                                       const Network::Connection *conn,
-                                       const Http::HeaderMap &headers,
-                                       const Http::AccessLog::RequestInfo &info,
-                                       const Router::RouteEntry *route) {
+void AccessLog::Entry::InitFromRequest(
+    std::string listener_id, const Network::Connection *conn,
+    const Http::HeaderMap &headers, const Envoy::AccessLog::RequestInfo &info,
+    const Router::RouteEntry *route) {
```

Comment:
These were aligned before, and easier to read. I'm not sure if it was intentional to change the formatting, if so, ignore me!